docs/modules/ROOT/pages/usage.adoc

```diff
@@ -242,26 +240,90 @@ Druid can use HDFS as a backend for deep storage:
 [source,yaml]
 ----
 spec:
-  deepStorage:
-    hdfs:
-      configMapName: simple-hdfs # <1>
-      directory: /druid # <2>
+  clusterConfig:
+    deepStorage:
+      hdfs:
+        configMapName: simple-hdfs # <1>
+        directory: /druid # <2>
 ...
 ----
 <1> Name of the HDFS cluster discovery config map. Can be supplied manually for a cluster not provided by Stackable. Needs to contain the `core-site.xml` and `hdfs-site.xml`.
 <2> The directory where to store the druid data.
 
-== Using Open Policy Agent (OPA) for Authorization
+== Security
+
+The Druid cluster can be secured and protected in multiple ways.
+
+=== Encryption
+
+TLS encryption is supported for internal cluster communication (e.g. between Broker and Coordinator) as well as for external communication (e.g. between the Browser and the Router Web UI).
+
+[source,yaml]
+----
+spec:
+  clusterConfig:
+    tls:
+      secretClass: tls # <1>
+----
+<1> Name of the `SecretClass` that is used to encrypt communication.
+
+IMPORTANT: A Stackable Druid cluster is always encrypted per default. In order to disable this default behavior you can set `spec.clusterConfig.tls: null`.
+
+=== Authentication
+
+==== TLS
+
+The access to the Druid cluster can be limited by configuring client authentication (mutual TLS) for all participants. This means that clients (e.g. the Browser or a Broker) have to authenticate themselves with valid certificates in order to communicate with the Druid cluster.
```

Comment:
It's not clear if this also enables internal tls communication between Druid servers or not. From the Druid docs, I understand that TLS is an all or nothing option. Also the operator code uses `tls` and `authentication` interchangeably. More clarification is needed.

Reply:
Tried to improve it here https://github.com/stackabletech/druid-operator/pull/333/commits/d0ab6ab77bcc8c1ad5dfe8383ea578731b8b342e